index.html

```diff
@@ -1491,6 +1469,29 @@ <h5 id="General Traits">General Traits</h5>
                <td>Global Level 3 or Nature Level 1
                </td>
             </tr>
+            <tr>
+               <td>Purity of Body
+               </td>
+               <td>
+                  When you take a short rest you can regain 5 resistance points of your choice instead of healing.
```

Comment:
This seems ok, isn't it kinda what you wanted to remove though? Everyone would benefit from having this is they haven't lost much HP but have lost Resistance Points as they take a short rest, but it maybe adds an interesting choice between HP and resistance points? I can't quite work out how interesting.